sys-utils/libcmt/include/libcmt/abi.h

```diff
@@ -129,14 +129,42 @@ enum {
     (((uint32_t) (A) << 000) | ((uint32_t) (B) << 010) | ((uint32_t) (C) << 020) | ((uint32_t) (D) << 030))
 #endif
 
-// put section ---------------------------------------------------------------
+/** EVM address */
+typedef struct cmt_abi_address {
+    uint8_t data[CMT_ADDRESS_LENGTH];
+} cmt_abi_address_t;
+
+/** EVM u256 in big endian format */
+typedef struct cmt_abi_u256 {
+    uint8_t data[CMT_WORD_LENGTH];
```

Comment:
Chan we change CMT_WORD_LENGTH to CMT_ABI_U256_LENGTH?

Reply:
> Chan we change CMT_WORD_LENGTH to CMT_ABI_U256_LENGTH here?

sure

> And elsewhere, can we change it to CMT_HASH_LENGTH or CMT_KECCAC_LENGTH wherever that is the thing with the size?

There is a `CMT_KECCAK_LENGTH` already in keccak.h